src/main/java/twilightforest/block/TrophyPedestalBlock.java

```diff
@@ -126,7 +126,7 @@ private boolean areNearbyPlayersEligible(Level level, BlockPos pos) {
 	}
 
 	private boolean isPlayerEligible(Player player) {
-		return PlayerHelper.doesPlayerHaveRequiredAdvancements(player, TwilightForestMod.prefix("progress_lich"));
+		return PlayerHelper.doesPlayerHaveRequiredAdvancements(player, TwilightForestMod.prefix("progress_lich")) || player.getAbilities().instabuild;
```

Comment:
> Wouldn't Player#isCreative make more sense?

`player.getAbilities().instabuild` is not only consistent with vanilla, but its also the way we do it in most, if not all scenarios

Reply:
I feel like that's a legacy thing that hasn't really been addressed by vanilla.

Reply:
"I placed player.getAbilities().instabuild because it is the ability to build. If there were a mod that added creative mode with health points (or any other game mode with instabuild = true), it would return true (and that makes sense to me).

Reply:
Makes sense to me.